citus.spec

```diff
@@ -79,6 +79,9 @@ echo %{pginstdir}/share/extension/%{sname}.control >> installation_files.list
 %doc %{pginstdir}/doc/extension/README-%{sname}.md
 
 %changelog
+* Thu Jul 08 2021 - Gurkan <gindibay@microsoft.com> 9.4.5.citus-1
+- Official 9.4.5 release of Citus
+
 * Thu Mar 18 2021 - Gurkan Indibay <gindibay@microsoft.com> 10.0.3.citus-1
```

Comment:
We are not always consistent with our changelog contents when we release a backport.

Does it really make sense to have changelog items for 10.x on a 9.y package? I do not think so.

A solution is to create this branch right after the commit that created 9.4.4 packages. This way, it will not have any entries for 10.x. It will still contain 9.5.1 which is a bummer.

All in all, I am fine with this file now, and this is only because it will take so much effort and it is not worth fixing this IMO.

Reply:
There may be some all-citus-9, all-citus-10 and we can continue on this branch. After these releases we can plan such structure